### PR TITLE
[windows] split docker install into 3 scripts

### PR DESCRIPTION
**Pull request by ilia-shipitsin**
# Description

the idea is to end with 1 script per 1 component for better observability

#### Related issue:

https://github.com/github/c2c-actions-akvelon/issues/38

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Install-Docker-Compose.ps1 [hunk 1]
```diff
@@ -0,0 +1,16 @@
+################################################################################
+##  File:  Install-Docker-Compose.ps1
+##  Desc:  Install Docker Compose.
```
Comment (erik-bershel) [inline images/win/scripts/Installers/Install-Docker-Compose.ps1:3]:
```suggestion
##  Desc:  Install Docker Compose.
##  Supply chain security: Docker Compose v1 - by package manager
```

**Review by erik-bershel — APPROVED** [review]
Small addition

File: images/win/scripts/Installers/Install-Docker-WinCred.ps1 [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+{
+ Param (
+    [Parameter(Mandatory = $True)]
+    [string] $Release
+)
+
+ $hashURL = "https://github.com/docker/docker-credential-helpers/releases/download/${Release}/checksums.txt "
+ (Invoke-RestMethod -Uri $hashURL).ToString().Split("`n").Where({ $_ -ilike "*docker-credential-wincred-${Release}.windows-amd64.exe*" }).Split(' ')[0]
+
+}
```
Comment (vpolikarpov-akvelon) [inline images/win/scripts/Installers/Install-Docker-WinCred.ps1:18]: Let's follow style guide and fix indentations and braces placement.
```suggestion
function Get-DockerWincredHash {
    Param (
        [Parameter(Mandatory = $True)]
        [string] $Release
    )

    $hashURL = "https://github.com/docker/docker-credential-helpers/releases/download/${Release}/checksums.txt"
    (Invoke-RestMethod -Uri $hashURL).ToString().Split("`n").Where({ $_ -ilike "*docker-credential-wincred-${Release}.windows-amd64.exe*" }).Split(' ')[0]
}
```

File: images/win/scripts/Tests/Docker.Tests.ps1 [hunk 3]
```diff
@@ -1,25 +1,35 @@
 Describe "Docker" {
     It "<ToolName>" -TestCases @(
-        @{ ToolName = "docker" }
         @{ ToolName = "docker-compose" }
     ) {
         "$ToolName --version" | Should -ReturnZeroExitCode
     }
```
Comment (vpolikarpov-akvelon) [inline images/win/scripts/Tests/Docker.Tests.ps1:6]: Do you want to get rid of test cases list as only one element left?

File: images/win/scripts/Tests/Docker.Tests.ps1 [hunk 4]
```diff
@@ -1,25 +1,35 @@
...
+        "C:\Windows\SysWOW64\docker.exe ps" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "DockerCompose" {
+    It "<ToolName>" -TestCases @(
+        @{ ToolName = "docker-compose" }
+    ) {
+        "$ToolName --version" | Should -ReturnZeroExitCode
     }
```
Comment (vpolikarpov-akvelon) [inline images/win/scripts/Tests/Docker.Tests.ps1:22]: Do you want to get rid of test cases list as only one element left?

File: images/win/scripts/Tests/Docker.Tests.ps1 (hunk 3 above)
Comment (ilia-shipitsin) [reply images/win/scripts/Tests/Docker.Tests.ps1:6]: no

**Review by vpolikarpov-akvelon — APPROVED** [review]

**Pull request merged**